paths/release_triggers/show.yaml

```
@@ -0,0 +1,44 @@
+---
+summary: Get a single release trigger
+description: Get details on a single release trigger.
```

small typo I believe. 
```suggestion
description: Get details of a single release trigger.
```